cache/test_cache_control.py

```diff
@@ -74,7 +72,12 @@ def setUp(self):
         self.tempesta = copy.deepcopy(self.tempesta_template)
         self.tempesta['config'] = self.tempesta['config'] % \
                         {'tempesta_config': self.tempesta_config or ''}
-
+        if 'http_chain' not in self.tempesta['config']:
+            self.tempesta['config'] += """
+            http_chain {
+                -> default;
+            }
+            """
```

Comment:
Please don't update configuration in runtime - this makes the test more complex for analyzing